datadog_checks_dev/datadog_checks/dev/tooling/commands/clean.py

```diff
@@ -62,16 +65,21 @@ def clean(ctx, check, compiled_only, all_matches, force, verbose):
         if basepath(path) in ('integrations-core', 'integrations-extras'):
             if force:
                 force_clean_root = True
-            else:
+            elif not compiled_only:
                 echo_warning(
-                    'You are running this from the root of the integrations project. '
-                    'Should we remove everything, including: '
-                    '.cache, .coverage, .eggs, .pytest_cache, .tox, build, dist, and *.egg-info? '
-                    'You can also use --force or -f to bypass this input.'
+                    'You are running this from the root of the integrations project.\n'
+                    'By default, the following artifacts in the root directory will *not* be cleaned:'
+                )
+                echo_info('.cache, .coverage, .eggs, .pytest_cache, .tox, build, dist, and *.egg-info')
+                force_clean_root = click.confirm(
+                    'Should we clean the above artifacts too? (Use --force of -f to bypass this prompt)'
                 )
-                force_clean_root = click.confirm('Do you want to continue?')
+                if force_clean_root:
+                    target_description = 'all artifacts'
+                else:
+                    target_description = 'artifacts (excluding those listed above)'
 
-    echo_waiting('Cleaning `{}`...'.format(path))
+    echo_waiting('Cleaning {} in `{}`...'.format(target_description, path))
```

Comment:
Suggestion, what about moving the whole section into the else condition of `if compiled_only` since this block doesn't apply if we are not in `compiled_only` condition?

Example:

```python
    if check:
        ....
    else:
        path = os.getcwd()

    if compiled_only:
        echo_waiting('Cleaning compiled only `{}`...'.format(path))
        removed_paths = remove_compiled_scripts(path, detect_project=not all_matches)
    else:
        echo_waiting('Cleaning `{}`...'.format(path))
        if not check and basepath(path) in ('integrations-core', 'integrations-extras'):
            if force:
                force_clean_root = True
            else:
                echo_warning(
                    'You are running this from the root of the integrations project. '
                    'Should we remove everything, including: '
                    '.cache, .coverage, .eggs, .pytest_cache, .tox, build, dist, and *.egg-info? '
                    'You can also use --force or -f to bypass this input.'
                )
                force_clean_root = click.confirm('Do you want to continue?')
        removed_paths = clean_package(path, detect_project=not all_matches, force_clean_root=force_clean_root)

```